adjust_copyright.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
```

Comment:
please do not commit the shell file. is not needed and should not be part of the repository

Reply:
- I've moved it one level up into the parent directory